README.md

```diff
@@ -2,20 +2,38 @@
 
 ## Quickstart
 
+### Build and start the container
+
 ```sh
-# Build and start the container
 docker compose up -d
+```
 
-# Install Node modules
+### Install Node modules
+
+```sh
 npm install
+```
+
+### Create a static assets directory
 
-# Create a static assets directory
+```sh
 mkdir app/static/assets
+```
+
+### Copy in the TNA Frontend static assets
 
-# Copy in the TNA Frontend static assets
+```sh
 cp -r node_modules/@nationalarchives/frontend/nationalarchives/assets/* app/static/assets
 ```
```

Comment:
I don't know if it makes sense to separate these as they all have to be run for the first install

Reply:
When they are combined, on VS code its `Pasting 11 links of text to the terminal` I found when there's an error on one of them ex `mkdir app/static/assets`, the next command following that also executes. This might not be a problem here in this scenario, but it does give a feel to scroll up+down and check which all commands executed successfully or not and then copy and run only them.

Reply:
See https://github.com/nationalarchives/django-application-template/blob/main/README.md#quickstart for updated (and simpler) instructions